src/pages/home/sidebar/SidebarLinks.js

```diff
@@ -72,6 +87,12 @@ const propTypes = {
 };
 
 const defaultProps = {
+    title: 'Chats',
+    showAvatar: true,
+    showCloseButton: false,
+    onAvatarClick: () => console.error('`onAvatarClick` is not defined'),
+    onCloseButtonClick: () => console.error('`onCloseButtonClick` is not defined'),
```

Comment:
I like this pattern where we use a console.error if we are showing an element that doesn't have a callback, but still displaying it.

I think if that's the case, `showAvatar` should be `false` by default right? I think it would be fine if you changed the `<SidebarLinks` call in  `src/pages/home/sidebar/SidebarView.js`to include `showAvatar`.

Reply:
I was a bit afraid to interfere with default behavior, but you're probably right. It's cleaner and removes implicity.
I've updated the code.